src/types/index.ts

```diff
@@ -14,6 +14,7 @@ export enum NetworkName {
   Kusama = 'kusama',
   Westend = 'westend',
   AlephZeroTestnet = 'alephzerotestnet',
+  AzeroDevNet = 'azerodevnet',
```

Comment:
Turns out _devnet_ was missing from the networks definition and nothing was broken (nothing was broken _by accident_ - this refactore surfaced that problem) 😶 Same ol' story - types not matching runtime.

Reply:
<details>
<summary></summary>

> ![image](https://user-images.githubusercontent.com/6209244/233171917-b28003e0-ecb3-465f-9f70-a636ba38806f.png)

![image](https://user-images.githubusercontent.com/6209244/233171575-963b742a-2d55-4d5d-a11c-0fbfc3967677.png)
</details>